pkg/kube/client.go

```diff
@@ -122,10 +122,15 @@ func (c *Client) newBuilder() *resource.Builder {
 }
 
 // Build validates for Kubernetes objects and returns unstructured infos.
-func (c *Client) Build(reader io.Reader) (ResourceList, error) {
+func (c *Client) Build(reader io.Reader, validate bool) (ResourceList, error) {
+	schema, err := c.Factory.Validator(validate)
+	if err != nil {
+		return nil, err
+	}
 	result, err := c.newBuilder().
 		Unstructured().
 		Stream(reader, "").
+		Schema(schema).
```

Comment:
Should `Schema` be before `Stream` as per https://github.com/helm/helm/pull/5576 ?